MinPlatformPkg/MinPlatformPkg.dsc

```diff
@@ -190,6 +190,9 @@
 
   MinPlatformPkg/PlatformInit/ReportFv/ReportFvPei.inf
   MinPlatformPkg/PlatformInit/PlatformInitPei/PlatformInitPreMem.inf
+  # MU_CHANGE [BEGIN] For MU repo pipeline requirement
+  MinPlatformPkg/PlatformInit/PlatformInitPei/PlatformInitPreMemNonFsp.inf
+  # MU_CHANGE [END]

```

Comment:
This should be in the upstream code as well. Project Mu just enforces that all code is actually built in a package build.